tests/bluetooth/iso/modules/iso_broadcast_sink.c

```diff
@@ -0,0 +1,522 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
```

Comment:
```suggestion
 * Copyright (c) 2023 Nordic Semiconductor ASA
```

Reply:
This code was originally written in 2021 so this is the way it should be. Let me see if I can find the link to the reasoning behind this.